ntoskrnl/cc/fs.c

```diff
@@ -355,10 +356,15 @@ CcSetLogHandleForFile (
     IN PVOID LogHandle,
     IN PFLUSH_TO_LSN FlushToLsnRoutine)
 {
+    PROS_SHARED_CACHE_MAP Map = FileObject->SectionObjectPointer->SharedCacheMap;
+
     CCTRACE(CC_API_DEBUG, "FileObject=%p LogHandle=%p FlushToLsnRoutine=%p\n",
         FileObject, LogHandle, FlushToLsnRoutine);
 
-    UNIMPLEMENTED;
+    if (!Map) return;
+
+    Map->LogHandle = LogHandle;
```

Comment:
This seem to be correct, but does it make sense to add these variables to the structure, before we have a code which uses them?

Reply:
I think it does, because when we'll add that code, it will not needed to fix this implementation later.